docs/core/deploying/runtime-package-store.md

```diff
@@ -0,0 +1,80 @@
+
+The feature is also used implicitly by ASP.NET applications: the set of packages composing the ASP.NET Web framework is installed as part of the setup packages authored by Microsoft. When publishing an ASP.NET application, the published application is trimmed to only include the application's packages, and not the framework's packages.
+
+## Publishing an application against a target manifest
+
+If you have a target manifest file on disk, you can specify it when publishing your app with the [`dotnet publish`](../tools/dotnet-publish.md) command:
+
+```
+dotnet publish --manifest path/to/the/target-manifest.xml
+```
+
+The resulting published application should only be deployed to an environment that has the packages described in the target manifest. Failing to do so would result in the application not starting.
+
+It's possible to specify multiple target manifests when publishing an application. The application will then be trimmed for the union of packages specified in those target manifests.
+
+## Specifying a target manifest in the project file
+
+Instead of specifying a target manifest in a [`dotnet publish`](../tools/dotnet-publish.md) command, it's possible to specify the manifest or manifests to use in the project file as a semicolon-separated list of paths under a `TargetManifestFiles` tag.
+
+```xml
+<PropertyGroup>
+  <TargetManifestFiles>path/to/the/target-manifest.xml</TargetManifestFiles>
+</PropertyGroup>
+```
+
+This should only be done when the target environment for the application is well-known. This is the case for example for ASP.NET projects.
+
+A different case would be an open-source project: the users of the project will likely deploy to a variety of different production environments, which may have different sets of packages pre-installed. Maintainers of the project can't make assumptions about the target manifest, and users should instead rely on the `--manifest` option of [`dotnet publish`](../tools/dotnet-publish.md) instead.
+
+## ASP.NET implicit store
+
+The default ASP.NET targets included with the `Microsoft.NET.Sdk.Web` SDK include target manifests. As a consequence, `dotnet publish` of an ASP.NET application will result in a published application that contains only the application and its assets, and not ASP.NET itself.
+
+When an ASP.NET published application gets deployed, one should make sure that the target environment has ASP.NET installed, as the presence of .NET Core alone will not be sufficient.
+
+If the application needs to be published to such an environment that doesn't include ASP.NET, it is possible to opt out of the implicit store by specifying a `PublishWithAspNetCoreTargetManifest` flag set to false in the project file.
```

Comment:
what's the consequence of doing this? Does ASP.NET get included if you set this flag to false?